clearcut_detection_backend/frontend/src/App.js

```diff
@@ -131,17 +167,46 @@ class App extends Component {
             startDate,
             endDate
           });
+        })
+        .catch(() => {
+          alertMessage('Unable to load data', 'error');
+          this.setState({ loading: false });
         });
     } else {
       this.setState({ startDate, endDate });
     }
   }
 
   render() {
-    const { viewport, data, activePolygonData, startDate, endDate, focusedFilterInput, loading, activeItem, position } = this.state;
+    const {
+      viewport,
+      data,
+      activePolygonData,
+      startDate,
+      endDate,
+      focusedFilterInput,
+      loading,
+      activeItem,
+      position
+    } = this.state;
 
     return (
       <div className="App">
+        <Sidebar>
+          <About />
+          <Calendar
+            startDate={startDate}
+            endDate={endDate}
+            focusedInput={focusedFilterInput}
+            onDatesChange={this.onDatesChange}
+            onFocusChange={focusedInput =>
+              this.setState({ focusedFilterInput: focusedInput })
+            }
+          />
+          <CustomLegend data={CUSTOM_LEGEND_DATA} />
+          <Menu links={links} />
+        </Sidebar>
+
```

Comment:
when you open at first - it's normal, no problems, but when you change portrait orientation to landscape it has problem with map width, please handle this value and set coorrect with to map 

see pics. 

![image](https://user-images.githubusercontent.com/23707448/68471721-72218180-0227-11ea-81f2-ye16cd54b148c.png)

![image](https://user-images.githubusercontent.com/23707448/68471742-7cdc1680-0227-11ea-9399-275d54698d5e.png)